python-packages/fle_utils/settingshelper.py

```diff
@@ -68,9 +68,7 @@ def import_installed_app_settings(installed_apps, global_vars, cur_app="__root__
                 #logging.warn("(%s) %s is already set; resetting can cause confusion." % (app, var))
                 pass
 
-        #print "\n%s" % processed_apps
-        processed_apps = processed_apps.union(set([app]))
-        #print processed_apps
+        processed_apps.add(app)
```

Comment:
"I think the case here is that the way set.union works is by reference, and set.add works by value... well, that's my theory, I didn't look it up because it empirically just works for me."